docker-databases/prepare_db.bat

```diff
@@ -73,13 +63,4 @@ SET PROP_FILE=%HOMEPATH%\taskanaUnitTest.properties
     ECHO docker-compose -f %~dp0/docker-compose.yml down -v
     docker-compose -f %~dp0/docker-compose.yml down -v
     ECHO ---
-    GOTO REMOVE_PROP
```

Comment:
We probably also need to remove (or replace with "GOTO MENU") "GOTO REMOVE_PROP" in lines 59, 43 and 27. We also should remove "ECHO 6 - Remove %PROP_FILE%" in line 18 and "SET PROP_FILE=%HOMEPATH%\taskanaUnitTest.properties" in line 3.

Reply:
good catch. Done.